database/seedToPSQL.sql

```diff
@@ -0,0 +1,11 @@
+-- CREATE DATABASE apateez2;
```

Comment:
Is this file actually run when you are seeding, if you are just saving commands to be used in the terminal I would suggest putting them in your package.json or in your readme